src/lib/types/openapiDoc.ts

```diff
@@ -34,20 +34,343 @@ const document = {
+                schema: {
+                  type: 'object',
+                  properties: {
+                    data: {
+                      type: 'object',
+                      properties: {
+                        token: { type: 'string', pattern: 'rdme_\\w+' },
+                        label: { type: 'string', nullable: true },
+                        last_accessed_on: {
+                          type: 'string',
+                          format: 'date-time',
+                          nullable: true,
+                          description: 'An ISO 8601 formatted date for when the API key was last accessed.',
+                        },
+                        created_at: {
+                          type: 'string',
+                          format: 'date-time',
+                          description: 'An ISO 8601 formatted date for when the API key was created.',
+                        },
+                        uri: {
+                          type: 'string',
+                          pattern: '\\/projects\\/(me|[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*)\\/apikeys\\/[a-f\\d]{24}',
+                        },
+                      },
+                      required: ['token', 'label', 'last_accessed_on', 'created_at', 'uri'],
+                      additionalProperties: false,
+                    },
+                  },
+                  required: ['data'],
+                  additionalProperties: false,
+                },
+              },
+            },
+          },
+        },
+      },
+    },
     '/versions/{version}/apis': {
       get: {
         operationId: 'getAPIs',
         summary: 'Get all API definitions',
         tags: ['APIs'],
         description:
-          "Get all API definitions from your ReadMe project. \n\n>📘\n> This route is only available to projects that are using [ReadMe Refactored](https://docs.readme.com/main/docs/welcome-to-readme-refactored).\n\n>🚧 ReadMe's API v2 is currently in beta.\n> This API and its docs are a work in progress. While we don’t expect any major breaking changes, you may encounter occasional issues as we work toward a stable release. Make sure to [check out our API migration guide](https://docs.readme.com/main/reference/api-migration-guide), and [feel free to reach out](mailto:support@readme.io) if you have any questions or feedback!",
+          "Get all API definitions from your ReadMe project.\n\n>📘\n> This route is only available to projects that are using [ReadMe Refactored](https://docs.readme.com/main/docs/welcome-to-readme-refactored).\n\n>🚧 ReadMe's API v2 is currently in beta.\n >This API and its documentation are a work in progress. While we don't expect any major breaking changes, you may encounter occasional issues as we work toward a stable release. Make sure to [check out our API migration guide](https://docs.readme.com/main/reference/api-migration-guide), and [feel free to reach out](mailto:support@readme.io) if you have any questions or feedback!",
```

Comment:
😬 we'll need to double check that that callout change (e.g., `\n >This` renders properly in our docs